irods/connection.py

```diff
@@ -21,7 +21,7 @@
 
 
 ALLOW_PAM_LONG_TOKENS = True      # True to fix [#279]
-
+DESTRUCTOR_MSG = "connection __del__() called"
```

Comment:
Message to be logged when the connection destructor is called. Used in a unit test.